<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-G2P2MI3J" ignored="0" name="登录接口" tests="6" failures="4" timestamp="14 八月 2020 12:51:45 GMT" time="1.097" errors="0">
  <testcase name="test" time="0.143" classname="com.lemon.cases.LoginCase"/>
  <testcase name="test" time="0.141" classname="com.lemon.cases.LoginCase">
    <failure type="java.lang.AssertionError" message="expected [PASSED] but found [FAILED]">
      <![CDATA[java.lang.AssertionError: expected [PASSED] but found [FAILED]
	at com.lemon.cases.LoginCase.test(LoginCase.java:36)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.15" classname="com.lemon.cases.LoginCase">
    <failure type="java.lang.AssertionError" message="expected [PASSED] but found [FAILED]">
      <![CDATA[java.lang.AssertionError: expected [PASSED] but found [FAILED]
	at com.lemon.cases.LoginCase.test(LoginCase.java:36)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.133" classname="com.lemon.cases.LoginCase">
    <failure type="java.lang.AssertionError" message="expected [PASSED] but found [FAILED]">
      <![CDATA[java.lang.AssertionError: expected [PASSED] but found [FAILED]
	at com.lemon.cases.LoginCase.test(LoginCase.java:36)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.203" classname="com.lemon.cases.LoginCase">
    <failure type="java.lang.AssertionError" message="expected [PASSED] but found [FAILED]">
      <![CDATA[java.lang.AssertionError: expected [PASSED] but found [FAILED]
	at com.lemon.cases.LoginCase.test(LoginCase.java:36)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.195" classname="com.lemon.cases.LoginCase"/>
</testsuite> <!-- 登录接口 -->
